.github/workflows/precompiled.yaml

```diff
@@ -0,0 +1,84 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run this workflow on a schedule
+name: pre-compiled-image
+
+on:
+  schedule:
+    - cron: '00 09 * * *' # scheduled job
+
+jobs:
+  pre-compiled:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        driver:
+          - 535
+          - 550
+        flavor:
+          - aws
+          - azure
+          - generic
+          - nvidia
+          - oracle
+    steps:
+      - uses: actions/checkout@v4
+        name: Check out code
+      - name: Calculate build vars
+        id: vars
+        run: |
+          echo "COMMIT_SHORT_SHA=${GITHUB_SHA:0:8}" >> $GITHUB_ENV
+          echo "LOWERCASE_REPO_OWNER=$(echo "${GITHUB_REPOSITORY_OWNER}" | awk '{print tolower($0)}')" >> $GITHUB_ENV
+          REPO_FULL_NAME="${{ github.repository }}"
+          echo "LABEL_IMAGE_SOURCE=https://github.com/${REPO_FULL_NAME}" >> $GITHUB_ENV
+
+          GENERATE_ARTIFACTS="false"
```

Comment:
This needs to be "true" otherwise the images won't be pushed to the ghcr.io registry.